src/components/pages/pages.stories.js

```diff
@@ -111,9 +111,10 @@ function makeStory(name, themeSelector, disableChromatic = false) {
           <DialogFullScreen
             open={ store.get('open') }
             onCancel={ handleCancel }
+            pagesStyling
           >
             <PageState>
-              <DefaultPages
+              <Pages
                 pageIndex={ handleSlide(null, pageIndex()) }
               >
                 <Page title={ <Heading title='My First Page' /> }>
```

Comment:
Will `onCancel` prop not need to be passed as it's required in `IconButton`?

Reply:
There's where the issue with Pages that I mentioned is, it doesn't actually render the close button itself - the `DialogFullScreen` wrapping this has `onCancel` set and that's where it can be closed.
`Pages` doesn't even have an `onCancel` prop, which is part of what's going to have to change when this component is re-written IMO